artemis-jms-client/src/main/java/org/apache/activemq/artemis/jms/client/ActiveMQDestination.java

```diff
@@ -273,6 +286,14 @@ public void setAddress(String address) {
       setSimpleAddress(SimpleString.toSimpleString(address));
    }
 
+   public void setName(String name) {
```

Comment:
What's the use of these setters? 
They should not be used for serialization and I don't see why the name (or address) of an instantiated JMS destination would be changed.

Reply:
The address setter was added as part of an effort to make Artemis JNDI work properly in Tomcat.  I assume it's needed here, but I'll remove it for simplicity's sake and it can be added later if necessary.